repository/z.json

```diff
@@ -158,6 +158,16 @@
 				}
 			]
 		},
+		{
+			"name": "Zig Unofficial",
```

Comment:
Per your changes to the installation instructions and the `edit_settings` command, this should be:
```suggestion
			"name": "ZigUnofficial",
```